biodata-tools/src/main/java/org/opencb/biodata/tools/variant/VariantJsonReader.java

```diff
@@ -117,11 +117,28 @@ public boolean close() {
             i++;
         }
 
+        return normaliseIfappropriate(variants);
+    }
+
+    private List<Variant> normaliseIfappropriate(List<Variant> variants) {
+        // Need to normalise one by one so that if one of them raises error while normalising we can easily notify which
+        // one and skip it
+        List<Variant> finalVariantList;
         if (normalizer != null) {
-            variants = normalizer.apply(variants);
+            finalVariantList = new ArrayList<>(variants.size());
+            for (Variant variant : variants) {
+                try {
+                    finalVariantList.addAll(normalizer.apply(Collections.singletonList(variant)));
+                } catch (RuntimeException e) {
+                    logger.warn("Error found during variant normalization. Variant: {}. This variant will be skipped "
+                            + "and process will continue", variant.toString());
+                    e.printStackTrace();
```

Comment:
Avoid e.printStackTrace()
Use the logger instead, giving the exception as a parameter 
`logger.error("Error found during ..." , e)`